ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -156,6 +156,7 @@ echo "OO_PACKAGE:           $OO_PACKAGE"
 echo "OO_CHANNEL:           $OO_CHANNEL"
 echo "OO_INSTALL_NAMESPACE: $OO_INSTALL_NAMESPACE"
 echo "OO_TARGET_NAMESPACES: $OO_TARGET_NAMESPACES"
+echo "OO_CONFIG_ENVVARS:    $OO_CONFIG_ENVVARS"
```

Comment:
Should we add `EVAL_CONFIG_ENVVARS` to this logging output?

Reply:
Yes, I think so, I missed it.

Reply:
Done.